src/style.css

```diff
@@ -1,3 +1,70 @@
-body {
+* {
+  padding: 0;
   margin: 0;
 }
+
+body {
+  font-family: "Roboto", sans-serif;
+
+  --hover-color: #00acdc;
+}
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: block;
```

Comment:
```suggestion
```

Reply:
If display: block; is removed, the image gets additional space below it due to the base line of "a" element. So, this property has been added for correct vertical centering of image element.